tools/cldr-code/src/main/java/org/unicode/cldr/util/ExtraPaths.java

```diff
@@ -2,13 +2,71 @@
 
 import com.google.common.collect.ImmutableSet;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Set;
 import java.util.TreeSet;
+import org.unicode.cldr.test.CheckMetazones;
 
 public class ExtraPaths {
+    private static final boolean DEBUG = false;
 
-    public static void add(Collection<String> toAddTo) {
+    private static final SupplementalDataInfo supplementalData =
+            CLDRConfig.getInstance().getSupplementalDataInfo();
+
+    private static final ImmutableSet<String> casesNominativeOnly =
+            ImmutableSet.of(GrammarInfo.GrammaticalFeature.grammaticalCase.getDefault(null));
+
+    /**
+     * A set of paths to be added. These are constant across locales, and don't have good fallback
+     * values in root. NOTE: if this is changed, you'll need to modify
+     * TestPaths.extraPathAllowsNullValue
+     */
+    private static final Set<String> CONST_EXTRA_PATHS =
+            CharUtilities.internImmutableSet(
+                    Set.of(
+                            // Individual zone overrides
+                            "//ldml/dates/timeZoneNames/zone[@type=\"Pacific/Honolulu\"]/short/generic",
+                            "//ldml/dates/timeZoneNames/zone[@type=\"Pacific/Honolulu\"]/short/standard",
+                            "//ldml/dates/timeZoneNames/zone[@type=\"Pacific/Honolulu\"]/short/daylight",
+                            "//ldml/dates/timeZoneNames/zone[@type=\"Europe/Dublin\"]/long/daylight",
+                            "//ldml/dates/timeZoneNames/zone[@type=\"Europe/London\"]/long/daylight",
+                            "//ldml/dates/timeZoneNames/zone[@type=\"Etc/UTC\"]/long/standard",
+                            "//ldml/dates/timeZoneNames/zone[@type=\"Etc/UTC\"]/short/standard",
```

Comment:
Why are these static? seems extremely fragile. 

I know this isn't new and it's from refactoring, but how/why did we end up with a static list in the first place?

Reply:
These were items that the TC agreed to add.